Fix a broken link (#4430) (#4431)
pingcap/docs: best-practices/massive-regions-best-practices.md

>> pd-ctl config set merge-schedule-limit 8
```

> <<<<<<< HEAD

Reviewer: ```suggestion
```

> Refer to [Region Merge](https://github.com/tikv/tikv/blob/master/docs/how-to/configure/region-merge.md) for more details.

Reviewer: ```suggestion
Refer to [Region Merge](https://tikv.org/docs/4.0/tasks/configure/region-merge/) for more details.
```

> =======
> Refer to [Region Merge](https://tikv.org/docs/4.0/tasks/configure/region-merge/) and the following three configuration parameters in the [PD configuration file](/pd-configuration-file.md#schedule) for more details:
> 
> - [`max-merge-region-size`](/pd-configuration-file.md#max-merge-region-size)
> - [`max-merge-region-keys`](/pd-configuration-file.md#max-merge-region-keys)
> - [`merge-schedule-limit`](/pd-configuration-file.md#merge-schedule-limit)
> >>>>>>> 1ec0e92e... fix a broken link (#4430)

Reviewer: ```suggestion
```
